package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "1.170.4",
+      "version": "1.170.5",
```

Comment:
This change needs to be reverted in order to merge.
Edit it manually if you have to.

Reply:
I haven't made any changes in package-lock.json.

![Screenshot (41)](https://github.com/EddieHubCommunity/LinkFree/assets/113820860/98a76ec2-6746-4449-b26a-305a92901eda)

Reply:
git thinks you did.
We can't merge this until the version is changed back

Reply:
Is it okay now?

Reply:
Here is the diff. It shows what need to be changed:
![package-lock](https://github.com/EddieHubCommunity/LinkFree/assets/35927536/1c6e2772-2978-4332-8c5e-68151a60b09d)

Reply:
I don't know how this problem came in my files its showing 1.170.5 only. I will change it to 1.170.4 ,commit and again change it back to 1.170.5.

Reply:
Please use `npm ci` in the future not `npm install`, as this will generate a new lock file

You can revert the lock file with `git checkout main -- package-lock.json`

You can learn more about why and how here https://www.youtube.com/watch?v=jHkIXYVpMTo

Reply:
Got it